### ISSUE TITLE
Method Chaining in Objective C

### ISSUE THREAD

**Issue by emailatravi** [layoutBox/PinLayout#184]
Hi,

Can you provide methods chaining in Objective C as it is available in Swift.

**Comment by lucdion** [comment]
I agree @emailatravi, that would be really nice. It wouldn't be so complicated, it just needs time.
The implementation would be something similar to what Masonry does (https://github.com/SnapKit/Masonry).

**Comment by lucdion** [comment]
Very old topic. Closing it, sorry
